Alexa.NET/Response/Response.cs

```diff
@@ -17,12 +20,12 @@ public class ResponseBody
         [JsonProperty("shouldEndSession", NullValueHandling = NullValueHandling.Ignore)]
         public bool? ShouldEndSession { get; set; } = false;
 
-        [JsonProperty("directives", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonProperty("directives", NullValueHandling = NullValueHandling.Ignore, ItemConverterType = typeof(DirectiveConverter))]
```

Comment:
This should be on the IDirective definition itself, that way we have the advantage of this new deserialization regardless of where it's used or any future changes of ResponseBody

Reply:
Agreed I'll move the converter to IDirective instead